wagtailio/project_styleguide/templates/patterns/components/get-started/get-started.html

```diff
@@ -0,0 +1,13 @@
+<div class="get-started grid">
+    <h2 class="get-started__heading heading-one">{{ value.heading }}</h2>
+
+    {% if value.items.all %}
```

Comment:
Note: I'd double check your syntax here, `.all` is used in reference to querysets, here you are referencing a list of blocks so you should be able to just do `{% for item in value.items %}`

Reply:
thanks, updated!